sound/soc/sof/sof-of-dev.c

```diff
@@ -53,6 +84,10 @@ static int sof_of_probe(struct platform_device *pdev)
 	if (!sof_pdata)
 		return -ENOMEM;
 
+	sof_pdata->machine = devm_kzalloc(dev, sizeof(*sof_pdata->machine),
+					  GFP_KERNEL);
+	if (!sof_pdata->machine)
+		return -ENOMEM;
```

Comment:
ok, you re-purpose your double empty line here :-)

Reply:
@lyakh I understand now! Thanks for noticing this :). Will fix it in next version.